testing/test_debugging.py

```diff
@@ -913,8 +913,7 @@ def test_foo():
 class TestDebuggingBreakpoints:
     def test_supports_breakpoint_module_global(self) -> None:
         """Test that supports breakpoint global marks on Python 3.7+."""
-        if sys.version_info >= (3, 7):
-            assert SUPPORTS_BREAKPOINT_BUILTIN is True
+        assert SUPPORTS_BREAKPOINT_BUILTIN is True
```

Comment:
we can just factor out `SUPPORTS_BREAKPOINT_BUILTIN` since all versions support it